src/Maestro/FeedCleanerService/FeedCleanerService.cs

```diff
@@ -61,52 +61,64 @@ public Task<TimeSpan> RunAsync(CancellationToken cancellationToken)
     [CronSchedule("0 0 2 1/1 * ? *", TimeZones.PST)]
     public async Task CleanManagedFeedsAsync()
     {
-        if (Options.Enabled)
+        if (!Options.Enabled)
         {
-            Dictionary<string, Dictionary<string, HashSet<string>>> packagesInReleaseFeeds =
-                await GetPackagesForReleaseFeedsAsync();
+            Logger.LogInformation("Feed cleaner service is disabled in this environment");
+            return;
+        }
+
+        Logger.LogInformation("Loading packages in release feeds...");
+
+        Dictionary<string, Dictionary<string, HashSet<string>>> packagesInReleaseFeeds =
+            await GetPackagesForReleaseFeedsAsync();
+
+        Logger.LogInformation("Loaded {versionCount} versions of {packageCount} packages from {feedCount} release feeds",
+            packagesInReleaseFeeds.Sum(feed => feed.Value.Sum(package => package.Value.Count)),
+            packagesInReleaseFeeds.Sum(feed => feed.Value.Keys.Count),
+            packagesInReleaseFeeds.Keys.Count);
 
-            foreach (var azdoAccount in Options.AzdoAccounts)
+        foreach (var azdoAccount in Options.AzdoAccounts)
```

Comment:
it's not really an AzDo account tho, it's an org, maybe we should rename it accordingly

Reply:
Good point

Reply:
AzDO calls it organization but renaming this would need be quite a large change - it's across the whole AzDO client class

Reply:
let's leave it then